chat-commands.js

```diff
@@ -3239,26 +3235,26 @@ exports.commands = {
 		}
 
 		this.sendReply(`Fetching newest version...`);
-		logRoom.roomlog(`${user.name} used /updateserver`);
+		logRoom.roomlog(`${user.name} used /updateserver.`);
 
 		let [code, stdout, stderr] = await exec(`git fetch`);
-		if (code) throw new Error(`updateserver: Crash while fetching - make sure this is a Git repository`);
+		if (code) throw new Error(`updateserver: Crash while fetching - make sure this is a Git repository.`);
```

Comment:
Do error messages usually have periods in them? I don't think so....

Reply:
![image](https://user-images.githubusercontent.com/551184/39378271-7ed3f6b0-4a1d-11e8-85d2-4023bc21870f.png)

Yeah, they don't.